### PR TITLE
Newsletter 13: SIMple Physics

### PR DESCRIPTION
**Pull request by mkhan45**
For #232

### CODE REVIEW

File: content/posts/newsletter-013/index.md [hunk 1]
```diff
@@ -62,6 +62,41 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+- Extensibility
+
+[Find out more about the project here][simple-physics-site]
+
+[Watch some cool gifs here][simple-physics-gifs]
+
+[or find the GitHub page here][simple-physics-github]
+
+_Discussions:
+[/r/rust](https://www.reddit.com/r/rust/comments/ibk2rf/announcing_simple_physics_a_set_of_educational/)
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-013/index.md:93]: The italics markup isn't closed here:
```suggestion
[/r/rust](https://www.reddit.com/r/rust/comments/ibk2rf/announcing_simple_physics_a_set_of_educational/)_
```

**Review by 17cupsofcoffee — APPROVED** [review]
Looks alright to me, just missing one bit of markup on the discussions.

It might also be a little nicer to phrase the project/gifs/github links as a sentence, rather than having them broken over three lines?

**Comment by mkhan45** [comment]
> It might also be a little nicer to phrase the project/gifs/github links as a sentence, rather than having them broken over three lines?

I agree; I think it's better now

**Review by ozkriff — APPROVED** [review]
:+1: (made a few style/fmt tweaks and optimized the gif a little)

**Comment by ozkriff** [comment]
Thanks for the PR!

**Pull request merged**